games/o.yaml

```diff
@@ -2304,6 +2304,24 @@
   video:
     youtube: RYiUsDSApM8
 
+- name: OpenOP2
+  originals:
+  - Outpost 2
+  type: remake
+  repo: 'https://github.com/OpenOP2/OpenOP2'
+  feed: 'https://github.com/OpenOP2/OpenOP2/releases.atom'
+  development: sporadic
+  status: semi-playable
```

Comment:
how is the game semi playable? I can't see any indication of its playability status